packages/accounts/src/light-account/utils.ts

```diff
@@ -95,21 +196,66 @@ export const LightAccountUnsupported1271Factories = new Set(
   LightAccountUnsupported1271Impls.map((x) => x.factoryAddress)
 );
 
-export const getLightAccountVersion = async <A extends SmartContractAccount>(
+export type LightAccountSource = "LightAccount" | "MultiOwnerLightAccount";
```

Comment:
I don't think we should combine the two into one factory. Ideally we have two separate definitions -- one for LA and one for MOLA. We did the same for multi owner ma and multi sig ma

Reply:
for multi owner and multi sig, the code and functionalities of the accounts were vastly different, but for Light vs multiownerLight, the only difference is the ownership function. Yes I could have a light account base for the common light account source, but I think using light account type param is cleaner.